src/vscode-dts/vscode.proposed.remoteCodingAgents.d.ts

```diff
@@ -3,6 +3,27 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-// empty placeholder for coding agent contribution point from core
+declare module 'vscode' {
+	export interface RemoteCodingAgentInformation {
```

Comment:
Is any of this information static and could it be contributed in the package.json instead? This would be useful for avoiding activating extensions if we don't need to